cetmix_tower_server/models/cx_tower_server.py

```
@@ -1550,68 +1553,71 @@ def _run_python_code(
+            path (str, optional): Directory where the command should be executed.
+            sudo (str, optional): Sudo mode ('p' for password, 'n' for no password).
+            **kwargs: Additional keyword arguments.
+                Supported keys:
+                - no_split_for_sudo (bool): if True, disables splitting by '&&'.
 
         Returns:
-            command (list|str): command splitted into separate commands
-                (for sudo with password mode ('p')) or composed command
-                from its parts for sudo without password mode ('n'))
-
+            str | list[str]: Command string or list, depending on sudo mode and split options.
```

Ruff (0.8.2)

Line too long (94 > 88)

(E501)